.travis.yml

```diff
@@ -11,57 +11,14 @@ cache:
     - vendor/bundle
 
 rvm:
-  - "2.0.0"
-  - "2.3.8"
-  - "2.4.5"
```

Comment:
Are we dropping support for these rubies?

Reply:
No, but waiting an hour or two for every Ruby version I know works when I only need to test it against Ruby 2.5 (any older Ruby version) and 2.6 on TravisCI seems like a waste of time 😬 
I'm trying to narrow down a problem that only occurs on the TravisCI rubies, so that's why this approach for now.

Reply:
@tombruijn Ah yeah fair enough :D